.github/flavors.json

```diff
@@ -338,15 +338,5 @@
         "baseImage": "rockylinux:9",
         "arch": "amd64",
         "worker": "self-hosted"
-    },
-    {
-        "family": "nvidia",
```

Comment:
I removed it from the flavors because this one is no longer a family/distribution of its own but instead it uses the ubuntu family via the BYOI. See image-arm.yaml and release.yaml to see how the two work together